inference-engine/src/vpu/graph_transformer/src/stages/eltwise.cpp

```diff
@@ -167,7 +170,8 @@ class EltwiseStage final : public StageNode {
                              static_cast<Handle<StageNode>>(this), dataTypeInput1, supportedDataTypesInput1);
 
             assertInputsOutputsTypes(this, {{dataTypeInput0}, {dataTypeInput1}, {dataTypeInput1}}, {{dataTypeInput1}});
-        } else if (operation == StageType::Greater && dataTypeInput0 != dataTypeOutput) {
```

Comment:
Is there a specific test for the changed behavior?

Reply:
Yes, the layer tests with the appropriate precisions are actually targeted for this purpose (otherwise it woud be failed).

Reply:
What I was tried to say is that we didn't have such a test before, but now it is tested in a positive way via layer tests,